mmf/configs/defaults.yaml

```diff
@@ -220,3 +225,47 @@ distributed:
     # Set if you do not want spawn multiple processes even if
     # multiple GPUs are visible
     no_spawn: false
+
+# Configuration for checkpointing including resuming and loading pretrained models
+checkpoint:
+    # If checkpoint.resume is true, MMF will try to load automatically load
+    # checkpoint and state from "current.ckpt" from env.save_dir
+    resume: false
+    # `checkpoint.resume_file` can be used to load a specific checkpoint from a file
+    # Can also be a zoo key
+    resume_file: null
+    # `checkpoint.resume_best` will load the best checkpoint according to
+    # training.early_stop.criteria instead of the last saved ckpt
+    resume_bert: false
+    # `checkpoint.resume_pretrained` can be used in conjuction with `resume_file`
+    # or `resume_zoo` where you specify a checkpoint or .pth file to be loaded
+    # but it is mapped based on `checkpoint.pretrained_state_mapping`
+    resume_pretrained: false
+    # `checkpoint.resume_zoo` can be used to resume from a pretrained model provided
+    # in zoo. Value maps to key from zoo. `checkpoint.resume_file` has higher
+    # priority compared to `checkpoint.resume_zoo`.
+    resume_zoo: null
+    # `checkpoint.zoo_override` will override the current model config of trainer
+    # with what is provided from the zoo checkpoint and will load the model
+    # using .from_pretrained of the model passed
+    zoo_override: false
```

Comment:
Better to rename it as `zoo_config_override` for clarity.

Reply:
Updated.